src/components/Keyboard/Keyboard.jsx

```diff
@@ -0,0 +1,4380 @@
+import React from 'react'
+import styles from "./Keyboard.module.css";
+
+export default function Keyboard() {
+
+    function KeyText({ fill, symbolX, symbolY, children }) {
+        return (
+            <text
+                fill={ fill }
+                xmlSpace="preserve"
+                style={{ whiteSpace: "pre" }}
+                fontFamily="Open Sans"
+                fontSize="11"
+                fontWeight="600"
+                letterSpacing="0em"
+            >
+                <tspan x={symbolX} y={symbolY}>
+                    {children}
+                </tspan>
+            </text>
+        )
+    }
+
+    function Key ({ base, alt, shift, altShift, keyX, keyY }) {
+        return (
+            <g id="Key">
```

Comment:
You should not use same `id`. Maybe in fact you should not use `id` at all here.